crates/server/src/datastore/mod.rs

```diff
@@ -135,15 +131,16 @@ pub trait DataStore: Send + Sync {
         reason: &str,
     ) -> Result<(), DataStoreError>;
 
-    /// Validates the given operator record.
+    /// Commits the given operator record.
     ///
     /// The record must be in a pending state.
     ///
     /// If validation succeeds, the record will be considered part of the log.
-    async fn validate_operator_record(
+    async fn commit_operator_record(
```

Comment:
This is somewhat of an unrelated change but I think this name better captures that the method is expected to _update_ the store, not just validate.